lms/static/styles/frontend_apps.scss

```diff
@@ -24,22 +25,22 @@
 @use 'components/FileList';
 @use 'components/FilePickerApp';
 @use 'components/FullScreenSpinner';
-@use 'components/LMSFilePicker';
 @use 'components/LMSGrader';
 @use 'components/StudentSelector';
 @use 'components/SubmitGradeForm';
 @use 'components/ValidationMessage';
 @use 'components/SvgIcon';
 
-// Element styles
-@use 'variables' as var;
-
 // Make the shared package utility styles available for
 // initial prototyping purposes.
 // NB: These styles have not been made explicitly public by this package
 // yet, so this inclusion may be temporary.
 @use '@hypothesis/frontend-shared/styles/util' as hyp-u;
 
+@use 'variables' as var;
+
+// Element styles
+
```

Comment:
I just moved these because they felt like they were in the wrong spot in the module.